### ISSUE TITLE
Keyboard Navigation with Arrows does not work in Block Settings

### ISSUE THREAD

**Issue by ephox-mogran** [WordPress/gutenberg#3223]
## Issue Overview

Keyboard navigation does not work in the block settings dropdown, I'm not sure if there is an easy way to focus it at the moment. I'm currently using the mouse.

Note, the tab key works, but the arrow keys do not. Every time we don't capture the arrow keys, they propagate up to `WritingFlow` and there is a jarring block focus shift.

## Steps to Reproduce (for bugs)

1. Click on the block settings dropdown
2. Press the down button to focus the next item

## Expected Behavior
The next item in the menu should be focused

## Current Behavior
The menu is closed, and some other block is focused

## Possible Solution

Arrow keys need to be captured in a lot more places, or `WritingFlow` has to stop trying to handle them.

## Todos
- [ ] Tests
- [ ] Documentation

**Comment by aduth** [comment]
Closed in favor of #5245
